src/wx/xrc/SpeedupConfig.xrc

```diff
@@ -24,49 +24,55 @@
               <orient>wxHORIZONTAL</orient>
               <object class="sizeritem">
                 <object class="wxStaticText">
-                  <label>% of normal:</label>
+                  <label># of frames:</label>
                   <size>60,-1d</size>
                 </object>
                 <flag>wxALL|wxALIGN_CENTRE_VERTICAL</flag>
                 <border>5</border>
               </object>
               <object class="sizeritem">
-                <object class="wxSpinCtrl" name="SpeedupThrottle">
-                  <value>0</value>
+                <object class="wxSpinCtrl" name="SpeedupFrameSkip">
+                  <value>9</value>
                   <min>0</min>
-                  <max>600</max>
+                  <max>30</max>
                 </object>
                 <flag>wxALL|wxALIGN_CENTRE_VERTICAL</flag>
                 <border>5</border>
               </object>
               <object class="sizeritem">
-                <object class="wxChoice" name="SpeedupThrottleSel">
+                <object class="wxChoice" name="SpeedupFrameSkipSel">
                   <content>
                     <item>None</item>
-                    <item>25%</item>
-                    <item>50%</item>
-                    <item>75%</item>
-                    <item>100%</item>
-                    <item>125%</item>
-                    <item>150%</item>
-                    <item>175%</item>
-                    <item>200%</item>
-                    <item>225%</item>
-                    <item>250%</item>
-                    <item>275%</item>
-                    <item>300%</item>
-                    <item>325%</item>
-                    <item>350%</item>
-                    <item>375%</item>
-                    <item>400%</item>
-                    <item>425%</item>
-                    <item>450%</item>
-                    <item>475%</item>
-                    <item>500%</item>
-                    <item>525%</item>
-                    <item>550%</item>
-                    <item>575%</item>
-                    <item>600%</item>
+                    <item>1</item>
+                    <item>2</item>
+                    <item>3</item>
+                    <item>4</item>
+                    <item>5</item>
+                    <item>6</item>
+                    <item>7</item>
+                    <item>8</item>
+                    <item>9</item>
```

Comment:
9 frame skip means 1000% speedup... I don't think it is even playable beyond this.

Reply:
The way this works currently, is if frameskip is selected that is the number of frames to skip when speedup is in effect. When speedup is in effect, the audio driver never waits and just drops excess data. So skipping more frames just makes speedup go faster by reducing rendering time.